.github/workflows/ci.yaml

```diff
@@ -1,6 +1,7 @@
+---
 name: ci
 
-on:
+on:  # yamllint disable-line rule:truthy
```

Comment:
:wrench: I think IFF it's amenable this suggestion can be done in a follow on PR.
With the number of  "disable ... rule:truthy" in this PR it feels like we might consider using a [.yamllint](https://yamllint.readthedocs.io/en/stable/configuration.html) config file.
And either add `on` to the [allowed values](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy) or we disable `check-keys`.

Unfortunately it looks like missing values in `.yamllint` disables checks so one needs to start with the full [default config](https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration).

Reply:
Yeah, unfortunately this is "github is doing things differently from 'standard' [lulz] YAML" thing---so I actually want this lint in basically everything _except_ github actions... :-(

Reply:
we could drop a yamllint in just the .github actions folder with a comment inside saying why

Reply:
Gotcha, yeah, will do.

Reply:
Unfortunately, a yamllint file in `/.github/workflows/` didn't seem to take, and I'm also not super keen on disabling truthiness for all the files there---part of my expectation is that values expected to be true/false would be checked.

Instead I removed the linting hints and changed `on:` to `"on":` instead, which makes it a string, rather than a truthy value.